include/charmlite/core/message.hpp

```diff
@@ -78,6 +78,7 @@ namespace cmk {
         static constexpr auto has_collection_kind_ = has_continuation_ + 1;
         static constexpr auto is_packed_ = has_collection_kind_ + 1;
         static constexpr auto for_collection_ = is_packed_ + 1;
+        static constexpr auto for_location_update_ = for_collection_ + 1;
```

Comment:
`for_collection` is intended to encapsulate this behavior; please do not reserve a separate flag.